DOL.WHD.Section14c.Web/src/modules/app.js

```diff
@@ -82,8 +82,9 @@ require('./pages')(app);
 require('./services')(app);
 app.config(function(IdleProvider, KeepaliveProvider) {
   // configure Idle settings
-  IdleProvider.idle(780); // in seconds
+  IdleProvider.idle(10); // in seconds
```

Comment:
Is this really supposed to be 10 seconds now instead of 13 minutes?

Reply:
good catch. forgot to change this back